monkey-runner/README.md

```diff
@@ -41,6 +44,7 @@ buildscript {
 
 ...
 
+apply plugin: 'com.novoda.android-command'
```

Comment:
actually, do we really need to add the command plugin to the android app's (sample) classpath?